src/kakarot/errors.cairo

```diff
@@ -499,4 +499,67 @@ namespace Errors {
         dw 'a';
         dw 's';
     }
+
+    func precompileInputError() -> (error_len: felt, error: felt*) {
```

Comment:
should the error take as input the precompile number and append it to `error`?

Reply:
And well, I've guessed that this logic was not worth it